src/_app.ts

```diff
@@ -1,14 +1,23 @@
 import type { App } from 'vue'
-import { resolvePath } from '@/utils/helpers.js'
+import { resolvePath, imgPathToSrcSet } from '@/utils/helpers.js'
 
-const updateAttr = (attrName, el, relPath) => {
-  el.setAttribute(attrName, resolvePath(relPath))
+const setResolvedPathToAttr = (attrName: string, el: HTMLElement, relPath: string) => {
+  el.setAttribute(attrName, resolvePath(relPath) || '')
+  console.log('!@# output path: ', resolvePath(relPath) || '')
 }
 
 export default (app: App) => {
   // custom-directives
-  app.directive('src', (el, binding) => updateAttr('src', el, binding.value))
-  app.directive('href', (el, binding) => updateAttr('href', el, binding.value))
+  app.directive('src', (el, binding) => {
+    setResolvedPathToAttr('src', el, binding.value)
+  })
+  app.directive('srcset', (el, binding) => {
+    el.setAttribute(
+      'srcset',
+      imgPathToSrcSet(binding.value, parseInt(binding.arg || '1'))
```

Comment:
As far as naming conventions go, my understanding is that most people use this convention: `image.jpg` and `image@2x.jpg`.

One of the earliest libraries to automatically and conveniently load these images is [retina.js](https://imulus.github.io/retinajs/). That's the format it uses.

I asked Claude, "what is the standard method of naming image files for high dpi displays and including them in srcset?"

And this is what it said:

![Screenshot 2024-10-22 at 1 21 39 PM](https://github.com/user-attachments/assets/b3ef68b9-29e3-4f0f-86c8-5fa1d562a913)